toonz/sources/translations/german/colorfx.ts

```diff
@@ -0,0 +1,723 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="de_DE">
+<context>
+    <name>ArtisticSolidColor</name>
+    <message>
+        <source>Irregular</source>
+        <translation>Unregelmäßigkeit</translation>
+    </message>
+    <message>
+        <source>Horiz Offset</source>
+        <translation>Horizontaler Offset</translation>
+    </message>
+    <message>
+        <source>Vert Offset</source>
+        <translation>Vertikaler Offset</translation>
+    </message>
+    <message>
+        <source>Noise</source>
+        <translation>Rauschen</translation>
+    </message>
+</context>
+<context>
+    <name>MovingSolidColor</name>
+    <message>
+        <source>Offset</source>
+        <translation>Offset</translation>
```

Comment:
Depends which is more in common use. It would be good to get some feedback from German animators about this